### PR TITLE
test(custom profile): Add support for upgrade and longevity tests + hybrid-RAID

### PR DESCRIPTION
**Pull request by yarongilor**
Supports custom-profile specific disk configuration and CQL schema.
	A new longevity test configures a hybrid-RAID and runs a Gemini load of custom schema.
        A new upgrade test pre-loads custom dataset by initial load.
	The test runs additional custom profile load during the upgrade/rollback.
	It verifies the basic dataset after all nodes are upgraded.
Task: https://github.com/scylladb/qa-tasks/issues/21
[Test plan](https://docs.google.com/document/d/1omGkONrwtm165_8fWf8r8-_gMl9753TW9uII8P4tjDI/edit?usp=sharing)
(not ready for review)
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
[Tested](https://argus.scylladb.com/test/426a8005-5a6b-4b9c-8393-4b8df741a514/runs?additionalRuns[]=434730f3-a6ad-4e41-8df9-8c21e2e72114) and found one [issue](https://github.com/scylladb/scylla-enterprise/issues/3514)

File: data_dir/disk_conf/hybrid_raid.py [hunk 1]
```diff
@@ -0,0 +1,784 @@
+#!/usr/bin/env python3
+"""
+This script ensures that the system is in a state in which scylla can successfully start.
+Such logic includes:
+  * Verifying the filesystem on top of NVMe RAID and/or a single pd-ssd
+  * Re-configuring mounts to match the state of the system
+  * Configuring scylla to replace itself if this node lost its data
+"""
+import argparse
```
Comment (fruch) [inline data_dir/disk_conf/hybrid_raid.py:9]: @yarongilor where this script is coming from ?

File: test-cases/upgrades/customer-profile/rolling-upgrade-custom-d1.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# To be used in keyspace_custom_preload:
+prepare_cs_user_profiles:
+    - scylla-qa-internal/custom_d1/stress_before_upgrade.yaml
+#stress_before_upgrade: cassandra-stress user no-warmup profile=/tmp/stress_before_upgrade.yaml  ops'(insert=1)' cl=ALL n=5000000 -mode cql3 native -rate threads=1000 -pop seq=1..5000000
+
+n_db_nodes: 4
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.2xlarge'
```
Comment (fruch) [inline test-cases/upgrades/customer-profile/rolling-upgrade-custom-d1.yaml:21]: we need this running on GCP, so we should reference the correct types for that

File: data_dir/disk_conf/hybrid_raid.py (hunk 1 above)
Comment (yarongilor) [reply data_dir/disk_conf/hybrid_raid.py:9]: https://github.com/scylladb/scylla-qa-internal/pull/27

**Comment by yarongilor** [comment]
Have to retest https://github.com/scylladb/scylla-enterprise/issues/3514 with a stronger instance, since it failed for memory issue.

**Pull request closed without merging**